lib/screens/home_screen.dart

```diff
@@ -58,4 +59,52 @@ class _HomeScreenState extends State<HomeScreen> {
       ),
     );
   }
+
+  void _openQrCodeDialog(BuildContext context) {
+    if (kIsWeb) return;
+    final peerService =
+        Provider.of<ChangeCallbackNotifier<PeerService>>(context, listen: false)
+            .callbackProvider;
+    if (!peerService.isServerRunning) {
+      showDialog(
```

Comment:
This is a relatively long method, which moves the else statement "out of sight" from the if.
Please move the showDialog into a separate method and call this method in the if block instead.
This increases:

-  readability (because the if condition is only separated from the else by a single line)
- testability (because the method building the dialog con be tested on its own